docs/changelog/1765.md

```diff
@@ -0,0 +1 @@
+- Reduced the runtime of readData by optimizing BSpline interpolation.
```

Comment:
Optimizations can be many things :wink:
Interesting from a user-perspective is that these changes reduce the cost of subsequent evaluations as they construct the B-Spline only once, which makes higher-order schemes more cost-effective.